File: mapbox_maps_flutter_mobile/example/android/app/src/androidTest/java/com/mapbox/maps/mapbox_maps_example/MainActivityTest.java
package com.mapbox.maps.mapbox_maps_example;

import android.Manifest.permission;
import androidx.test.rule.ActivityTestRule;
import androidx.test.rule.GrantPermissionRule;
import dev.flutter.plugins.integration_test.FlutterTestRunner;
import org.junit.Rule;
import org.junit.runner.RunWith;

@RunWith(FlutterTestRunner.class)
public class MainActivityTest {

  @Rule
  public GrantPermissionRule permissionRule =
      GrantPermissionRule.grant(permission.ACCESS_COARSE_LOCATION);

  @Rule
  public ActivityTestRule<MainActivity> rule =
      new ActivityTestRule<>(MainActivity.class, true, false);
}

File: example/android/app/src/androidTest/mapbox_maps_example/MainActivityTest.kt
package com.mapbox.maps.mapbox_maps_example

import androidx.test.rule.ActivityTestRule
import dev.flutter.plugins.integration_test.FlutterTestRunner
import org.junit.Rule
import org.junit.runner.RunWith


@RunWith(FlutterTestRunner::class)
class MainActivityTest {
    @Rule
    var rule: ActivityTestRule<MainActivity> =
        ActivityTestRule(MainActivity::class.java, true, false)
}